### PR TITLE
Chore/tracefoss 1423 support semantic model batch provided by irs job response

### PR DESCRIPTION
**Pull request by mkanal**

### CODE REVIEW

**Review by ds-ext-sceronik — COMMENTED** [review]
Just a question regarding SerialPartTypization if it is correct due to past changes. other than. that looks good

File: docs/src/uml-diagrams/concept/TRACEFOSS-1423/domain-model.puml [hunk 1]
```diff
@@ -29,34 +28,50 @@ entity Relation {
        idShort : String
 }
 
+enum SemanticModelType {
+    SERIALPARTTYPIZATION,
```
Comment (mkanal) [inline docs/src/uml-diagrams/concept/TRACEFOSS-1423/domain-model.puml:32]:
```suggestion
    SERIALPART,
```

File: docs/src/uml-diagrams/concept/TRACEFOSS-1423/domain-model.puml [hunk 2]
```diff
@@ -29,34 +28,50 @@ entity Relation {
...
 }
 
-note right
-    SerialPart and Batch
-end note
-
-enum SemanticDataModel {
-    BATCH,
-    SERIALPART
+entity SerialPartTypization {
```
Comment (mkanal) [inline docs/src/uml-diagrams/concept/TRACEFOSS-1423/domain-model.puml:46]:
```suggestion
entity SerialPart {
```

File: docs/src/uml-diagrams/concept/TRACEFOSS-1423/domain-model.puml [hunk 3]
```diff
@@ -29,34 +28,50 @@ entity Relation {
...
 
-Asset --> SemanticDataModel
-Asset <|.. AsBuiltAsset
-Asset <|.. AsPlannedAsset
+
+SemanticModelType --> Asset
+Asset <|.. AsBuiltAspectModel
+Asset <|.. AsPlannedAspectModel
+AsBuiltAspectModel <|.. Batch
+AsBuiltAspectModel <|.. SerialPartTypization
```
Comment (mkanal) [inline docs/src/uml-diagrams/concept/TRACEFOSS-1423/domain-model.puml:72]:
```suggestion
AsBuiltAspectModel <|.. SerialPart
```

File: docs/src/uml-diagrams/concept/TRACEFOSS-1423/domain-model.puml [hunk 4]
```diff
@@ -29,34 +28,50 @@ entity Relation {
...
+AsPlannedAspectModel <|.. Batch
+AsPlannedAspectModel <|.. SerialPartTypization
```
Comment (mkanal) [inline docs/src/uml-diagrams/concept/TRACEFOSS-1423/domain-model.puml:74]:
```suggestion
AsPlannedAspectModel <|.. SerialPart
```

Comment (mkanal) [reply docs/src/uml-diagrams/concept/TRACEFOSS-1423/domain-model.puml:74]: AsPlannedAspectModel  and serialPart that does not match. Please check

File: docs/src/uml-diagrams/concept/TRACEFOSS-1423/domain-model.puml [hunk 5]
```diff
@@ -29,34 +28,50 @@ entity Relation {
...
-Asset --> SemanticDataModel
-Asset <|.. AsBuiltAsset
-Asset <|.. AsPlannedAsset
+
+SemanticModelType --> Asset
+Asset <|.. AsBuiltAspectModel
+Asset <|.. AsPlannedAspectModel
+AsBuiltAspectModel <|.. Batch
+AsBuiltAspectModel <|.. SerialPartTypization
+AsPlannedAspectModel <|.. Batch
```
Comment (mkanal) [inline docs/src/uml-diagrams/concept/TRACEFOSS-1423/domain-model.puml:73]: AsPlanned and Batch might not match please check

File: docs/src/uml-diagrams/concept/TRACEFOSS-1423/domain-model.puml (hunk 1 above)
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/src/uml-diagrams/concept/TRACEFOSS-1423/domain-model.puml:32]: SERIALPART correct

**Review by ds-mwesener — APPROVED** [review]

**Pull request merged**